blocks/color-palette/index.js

```diff
@@ -59,9 +59,13 @@ export function ColorPalette( { colors, disableCustomColors = false, value, onCh
 					renderContent={ () => (
 						<ChromePicker
 							color={ value }
-							onChangeComplete={ ( color ) => onChange( color.hex ) }
+							// onChangeComplete={ ( color ) => onChange( color.hex ) }
+							onChangeComplete={ ( color ) => {
+								const colorString = color.rgb.a === 1 ? color.hex : sprintf( 'rgba(%s,%s,%s,%s)', color.rgb.r, color.rgb.g, color.rgb.b, color.rgb.a );
```

Comment:
Unlike in PHP, `sprintf` is not a native function, and could have performance implications. While this is not a function we would expect to be called with any frequency, I'd rather we avoided become too comfortable with its use where simple concatenation would suffix.

You could, for example, take advantage of `Array#join`'s default argument being a comma:

```js
let colorString;
if ( color.rgb.a === 1 ) {
	colorString = color.hex;
} else {
	const { r, g, b, a } = color.rgb;
	colorString = `rgba(${ [ r, g, b, a ].join() })`;
}
```

(Or be explicit with `.join( ',' )`)